CHANGELOG.md

```diff
@@ -4,6 +4,47 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.15](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.15) - 2021-08-TBD
+
+Zebra's latest alpha contains the last of the changes to zebra chain state ahead of NU5 testnet activation and before concentrating on mempool work. The remaining NU5 validation will be completed prior to NU5 mainnet activation.
+
+### Added
+
+#### Network Upgrade 5
+
+- ZIP-209: Prohibit Negative Shielded Chain Value Pool Balances Implementation (#2546, #2554, #2561, #2566, #2569, #2576, #2577, #2578, #2596)
+- ZIP-221: FlyClient - Consensus-Layer Changes Implementation(#2531, #2553, #2583)
+
+### Changed
+
+- Cache note commitment tree roots to speed up chain synchronisation  (#2584)
+- Update state cache version to v9 (#2598)
+- Refactor HistoryTree type into NonEmptyHistoryTree and HistoryTree types (#2582)
+
+#### Testing and CI
+
+- Reduce number of nullifier property test cases (#2574)
+- Ensure `regenerate-stateful-test-disks.yml` syncs to the latest network upgrade (#2601)
+- Add a random number of transactions to generated blocks, rather than always 2 (#2567)
+- Increase coverage for generated chains and proptests (#2540)
+- Remove unreliable tests for generated chains (#2548)
+- Add test-only methods for modifying orchard shielded data and joinsplits (#2580)
+- Generate test chains with valid chain value pools (#2597)
+- Increase timeout of cached database creation (#2581)
+- Use fixed genesis coinbase data in generated genesis blocks (#2568)
+- Generate valid sapling shielded data for property tests (#2579)
+- Make `zebra-state` compile successfully by itself (#2611)
+- Optimize build to regenerate the test state faster (#2552)
+
+### Fixed
+
+- Reject connections from outdated peers after network upgrade activation (#2519)
```

Comment:
This is a requirement from ZIP-201, so I think it belongs in "Added":
https://zips.z.cash/zip-0201#rejecting-pre-overwinter-connections

Reply:
moved in f467eae